docs/en/operation/Metrics-System.md

```diff
@@ -65,6 +65,16 @@ $ curl <WORKER_HOSTNAME>:<WORKER_WEB_PORT>/metrics/json
 $ curl 127.0.0.1:30000/metrics/json/
 ``` 
 
+Send an HTTP request to `/metrics/json/` to fuse web port to get fuse metrics.
```

Comment:
I synced with @ssz1997 about updating the Alluxio POSIX API about enabling the fuse web in standalone fuse process.

After he created the PR and merged, can you help link to that doc and mentioning
if alluxio fuse web port is enabled by alluxio.fuse.web.enabled=true, send an HTTP request to `/metrics/json/` to fuse web port to get fuse metrics.

Reply:
@LuQQiu should be good now, PTAL